raiden/network/rpc/client.py

```diff
@@ -303,7 +303,7 @@ def _gaslimit(self, location='pending') -> int:
         gas_limit = quantity_decoder(last_block['gasLimit'])
         # The gas limit can fluctuate from the actual pending limit by a maximum
         # of up to a 1/1024th of the previous gas limit
-        return gas_limit - int(gas_limit / 1024)
+        return int((gas_limit - int(gas_limit / 1024)) * 0.8)
```

Comment:
what about floor division?
`gas_limit - gas_limit // 8`

Reply:
ah good point. Forgot we got that in python.

Reply:
I guess I can also remove the subtraction by 1/1024th as it is not gonna make a different since we get 80% of it anyway